panel/models/ace.ts

```diff
@@ -68,7 +68,8 @@ export class AcePlotView extends HTMLBoxView {
     this._update_language()
     this._editor.setReadOnly(this.model.readonly)
     this._editor.setShowPrintMargin(this.model.print_margin)
-    this._editor.on("change", () => this._update_code_from_editor())
+    this._editor.on("blur", () => this._update_code_from_editor())
```

Comment:
This seems wrong, I don't think Ace intentionally triggers a blur event, e.g. see https://github.com/ajaxorg/ace/issues/3323

Reply:
I guess maybe it's fine but at minimum there should be a test checking that the event is triggered on unfocus and when pressing enter.

Reply:
From my understanding, a blur event occurs when an HTML element loses focus. I have added a test on onfocus and removed on enter.

Reply:
Yeah I was primarily confused because `blur` on enter didn't make sense to me. I think there should be a way to sync the `value` without having to unfocus the editor but not sure what that should be. Suppose Ctrl+S would be somewhat standard.

Reply:
Yeah, I was basing it off TextInput/TextAreaInput behavior. I think Ctrl+S works or Enter like ChatAreaInput.

Reply:
I'd be fine with enter actually, triggering an update on a newline seems reasonable.

Reply:
I'd be against Enter because it's code.

e.g.
```
def test():  # press enter here
```

This would trigger a syntax error. I'd be fine with like Ctrl+Enter though, but I do like Ctrl+S most.

Reply:
Okay, Ctrl+S is actually used by the browser to save webpage, so I went with Ctrl/Cmd+Enter.